game_event/views.py

```diff
@@ -46,7 +49,7 @@ def process_game_event_form(request):
 
 def game_events(request):
     game_events = GameEvent.objects.all()
-
+    game_events_with_weather = []
```

Comment:
Variables should be declared as closest to where they are used. In this case, it should be just before the loop.

Reply:
@yftacherzog Fixed.